app/templates/macros/page_macros.html

```diff
@@ -2,30 +2,37 @@
     <div id="editor-{{ editable_html_obj.editor_name }}" contenteditable="false">
         {{ editable_html_obj.value|safe }}
     </div>
-
     {% if current_user.is_admin() %}
-       <button class="ui button start-edit">
+        <button class="ui button start-edit">
```

Comment:
Oh ok i do not see that indentation here in my code though. 

https://cloud.githubusercontent.com/assets/16725465/20024489/547e5c2e-a2bd-11e6-9e5c-7c9be433e39c.png

Maybe its a different place or I just need to push my new file.